roles/kubernetes-apps/ingress_controller/cert_manager/defaults/main.yml

```diff
@@ -4,6 +4,9 @@ cert_manager_user: 1001
 cert_manager_tolerations: []
 cert_manager_affinity: {}
 cert_manager_nodeselector: {}
+cert_manager_dns_policy: "ClusterFirst"
```

Comment:
What is the default value on cert-manager side if Kubespray doesn't specify the policy?
ClusterFirst or None?

It would be nice to specify the same value as cert-manager's default from Kubespray also to avoid affecting users with this pull request change.

Reply:
😁If no DNS policy is specified, the default DNS policy is `ClusterFirst`, or `cert_manager_dns_policy: ""` will also work.

Reply:
Could you provide some URL of cert-manager to see what values are available for this option?
I could not find suitable site on my side TBH.

Reply:
😀This is described in the k8s documentation:
* [Pod's DNS Policy](https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/#pod-s-dns-policy)
* [Pod's DNS Config](https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/#pod-dns-config)

Reply:
Thank you so much for providing that :-)

```
Note: "Default" is not the default DNS policy. If dnsPolicy is not explicitly specified, then "ClusterFirst" is used.
```

That is really tricky ;-)
It is nice to specify ClusterFirst as the default on Kubespray side to follow the above.